src/system/RoutesModule/Api/Base/CacheApi.php

```diff
@@ -14,14 +14,14 @@
 
 use ModUtil;
 use UserUtil;
-use Zikula_AbstractApi;
+use Zikula_AbstractBase;
 use Zikula_View;
 use Zikula_View_Theme;
 
 /**
  * Cache api base class.
  */
-class CacheApi extends Zikula_AbstractApi
+class CacheApi extends Zikula_AbstractBase
```

Comment:
to me this seems odd. We've never extended directly from AbstractBase. It is a deprecated class anyway. if you are going to use a deprecated class, why not just use AbstractApi like before? If not that then why use anything?